episodes/04-built-in.md

```diff
@@ -259,6 +259,63 @@ NameError: name 'aege' is not defined
 
 - Fix syntax errors by reading the source and runtime errors by tracing execution.
 
+## Other ways to get help
+There are several other ways that people often get help when they are stuck with their Python code.
+
+* Search the internet: 
+  paste the last line of your error message or the word "python" and a short description of what you want to do into your favourite search engine 
+  and you will usually find several examples where other people have encountered the same problem and came looking for help.
+* [StackOverflow](https://stackoverflow.com/questions) can be particularly helpful for this: answers to questions are presented as a ranked thread ordered according to how useful other users found them to be.
+* **Take care:** copying and pasting code written by somebody else is risky unless you understand exactly what it is doing!
+* ask somebody "in the real world". 
+  If you have a colleague or friend with more expertise in Python than you have, show them the problem you are having and ask them for help.
+* Sometimes, simply the act of formulating your question can help you to identify what is going wrong.
```

Comment:
Isn't the process of talking aloud what helps that process?
```suggestion
* Sometimes, simply the act of formulating your question aloud can help you to identify what is going wrong.
```

Reply:
Thanks for the suggestion. I will used "articulating" as I hope that will be more inclusive of non-speaking community members.